optimum/bettertransformer/models/decoder_models.py

```diff
@@ -476,10 +475,6 @@ def project(hidden_states, proj_layer, key_value_states, past_key_value):
                 attn_weights, p=self.layer.dropout, training=self.layer.training
             )  # (batch_size, n_heads, seq_length, key_length)
 
-            # Mask heads if we want to
```

Comment:
Why these have been removed?

Reply:
This is not supported and checked for above

Reply:
Thanks makes sense!